pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
I think citing a version number from a roadmap/mailing-list sounds too manual work, since this has to be parsed by humans rather than machine reading a git tag. Package updating is something that is meant to be automated away, and shouldn't be a CAPTCHA/Turing test.